### PR TITLE
Feature/resolved state

### PR DESCRIPTION
**Pull request by the-nathan-smith**
Adds comment resolved info to threads & disables focus on threads which are resolved
Updates version number

### CODE REVIEW

File: src/components/CommentApp/main.scss [hunk 1]
```diff
@@ -43,7 +43,7 @@ $color-focus-outline: #ffbf47;
 $color-text-base: darken($color-white, 85);
 $color-text-input: darken($color-white, 90);
 
-$color-comment-separator: #e6e6e6;
+$color-comment-separator: #aeb7bd;
```
Comment (ColinBeeby-Developer) [inline src/components/CommentApp/main.scss:46]: Should this be $color_nhsuk-grey-3 ?

File: src/components/CommentApp/main.scss [hunk 2]
```diff
@@ -52,7 +52,7 @@ $color-state-absent: #ff8f11;
 $color-state-live-draft: #43b1b0;
 
 $color-box-background: $color-white;
-$color-box-border: $color-grey-3;
+$color-box-border: #aeb7bd;
```
Comment (ColinBeeby-Developer) [inline src/components/CommentApp/main.scss:55]: Should this be $color_nhsuk-grey-3 ?

File: src/components/CommentApp/main.scss (hunk 1 above)
Comment (the-nathan-smith) [reply src/components/CommentApp/main.scss]: Yep, I've added it in now 👍

File: src/components/CommentApp/main.scss (hunk 2 above)
Comment (the-nathan-smith) [reply src/components/CommentApp/main.scss]: Yep, I've added it in now 👍

**Review by ColinBeeby-Developer — APPROVED** [review]
All looks good to me.

**Pull request merged**